# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/media/test/test_mp3_broadcast.html
# Commit: f691cde9edaf
# Full Hash: f691cde9edaf7a11fc554bbda5aaf30893d80170
# Author: Paul Adenot <paul@paul.cx>
# Date: 2023-07-21 19:39:17
# Description:
#   Bug 1844741 - Consistently use remaining padding when trimming mp3. r=alwu
#   
#   Creating the file was made in the following way:
#   - Create a VBR file
#   - Modify the padding field so that it spans multiple packets (greater than 1152
# ==============================================================================

diff -r 3c7aa15fc461 -r f691cde9edaf dom/media/test/test_mp3_broadcast.html
--- a/dom/media/test/test_mp3_broadcast.html	Fri Jul 21 17:28:03 2023 +0000
+++ b/dom/media/test/test_mp3_broadcast.html	Fri Jul 21 17:55:32 2023 +0000
@@ -26,7 +26,7 @@
             // This scenario is typical in radio broadcast scenario, when the
             // live-stream has a pre-recorded prelude. The reported duration,
             // after "ended" has been received, is the duration of playback.
-            { src: "single-xing-header-no-content-length.mp3", duration: 11.030997 },
+            { src: "single-xing-header-no-content-length.mp3", duration: 11.050839},
         ];
         var audio = window.a;
         // Prevent ESLint error about top-level await
